test/object_test.go

```diff
@@ -141,8 +142,15 @@ func TestDefaultObjectStatus(t *testing.T) {
 }
 
 func TestObjectFileBasics(t *testing.T) {
+	// The filename is likely to be something like C:\xxx and we don't allow
```

Comment:
@derekcollison @scottf Want to bring this to your attention regarding ObjectStore (low priority, I know, but understand that this is not likely to work on Windows as long as user uses files like `C:\foo\bar`).

Reply:
I think this is fine now. The key encoding has changed from simple replacement to base64 url encoding of the filename for use as in a subject (key). You can see the change here: https://github.com/nats-io/nats.go/pull/883/files#diff-12229ea844f42fd705168380fb7870bcd7d66f0c94b53ef0efe8b9ca50f05ec7R284

After this PR is merged and incorporated into my branch I will update the test.